db/src/urls.rs

```diff
@@ -136,12 +145,31 @@ async fn store_markdown(
     let category_result = store_category(pool, markdown_result.markdown_id, &category).await?;
     println!("Category result: {category_result:?}");
 
-    Ok(markdown_result)
+    Ok((markdown_result, markdown_content))
```

Comment:
Doesn't `markdown_result` already have the content at `markdown_result.markdown_content`? Do we need to pass is down again?

Reply:
You correct! I thought I was suppose to call `markdown_content` as a different argument